src/Sulu/Bundle/ContentBundle/DependencyInjection/Configuration.php

```diff
@@ -38,25 +38,17 @@ public function getConfigTreeBuilder()
                         ->scalarNode('error_template')
                             ->example('ClientWebsiteBundle:Preview:error.html.twig')
                         ->end()
-                        ->arrayNode('fallback')
-                            ->addDefaultsIfNotSet()
-                            ->children()
-                                ->scalarNode('interval')
-                                    ->defaultValue('5000')
-                                ->end()
+                        ->scalarNode('mode')
+                            ->defaultValue('auto')
+                            ->validate()
+                                ->ifNotInArray(array('auto', 'on_request', 'off'))
+                                ->thenInvalid('Invalid preview mode "%s"')
                             ->end()
                         ->end()
-                        ->arrayNode('websocket')
-                            ->addDefaultsIfNotSet()
-                            ->children()
-                                ->scalarNode('port')
-                                    ->defaultValue('9876')
-                                ->end()
-                                ->scalarNode('url')
-                                    ->defaultValue('ws://sulu.lo')
-                                ->end()
-                            ->end()
+                        ->scalarNode('delay')
+                            ->defaultValue(200)
                         ->end()
+                        ->booleanNode('websocket')->defaultTrue()->end()
```

Comment:
Does anything additional have to be started for the websocket to work? If yes I would default to false, since not everybody will have a websocket running.